src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```
@@ -260,20 +261,17 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
      */
     protected function createConfigurationDefinition()
     {
-        $fixerName = $this->getName();
-
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('default', 'Default fix strategy.'))
                 ->setDefault(self::SINGLE_SPACE)
                 ->setAllowedValues(self::$allowedValues)
                 ->getOption(),
             (new FixerOptionBuilder('operators', 'Dictionary of `binary operator` => `fix strategy` values that differ from the default strategy.'))
                 ->setAllowedTypes(['array'])
-                ->setAllowedValues([function ($option) use ($fixerName) {
+                ->setAllowedValues([function ($option) {
                     foreach ($option as $operator => $value) {
                         if (!in_array($operator, self::$supportedOperators, true)) {
-                            throw new InvalidFixerConfigurationException(
-                                $fixerName,
+                            throw new InvalidOptionsException(
```

we shall throw this exception (we do it in other places), and then, in `AbstractFixer::configure`, it's converted to `InvalidFixerConfigurationException`